internal/services/fluidrelay/fluid_relay_servers_resource.go

```diff
@@ -256,6 +268,17 @@ func (s Server) Read() sdk.ResourceFunc {
 			if val, ok := meta.ResourceData.GetOk("storage_sku"); ok {
 				output.StorageSKU = val.(string)
 			}
+
+			keyRes, err := client.ListKeys(ctx, *id)
+			if err != nil {
+				// do not return if only list keys error
+				meta.Logger.Warnf("retrieving relay server keys err: %v", err)
```

Comment:
Could we change this to
```suggestion
				meta.Logger.Warnf("retrieving keys for %s: %v", *id, err)
```